src/UI/Settings/settings.less

```diff
@@ -100,6 +100,24 @@ li.save-and-add:hover {
   }
 }
 
+.proxy-setting {
+  display: none;
+
+  .control-label {
+    color: @brand-warning;
```

Comment:
No need to set the color, just need to set them to advanced and it happens automatically.